client/packages/invoices/src/Prescriptions/DetailView/Payments/PaymentsModal.tsx

```diff
@@ -23,11 +31,12 @@ export const PaymentsModal: FC<PaymentsModalProps> = ({
     query: { data },
   } = usePrescription();
 
-  const fields = [
+  const fields: PaymentModalField[] = [
     {
       label: t('label.total-to-be-paid'),
-      type: 'number',
       value: data?.pricing.totalAfterTax,
+      disabled: true,
+      onChange: noop,
```

Comment:
Isn't using 3rd party library a bit overkill? Why not just `onChange: () => {}` ?

Personally, I'd rather modify the CurrencyCell to make `onChange` optional, but we can save that for another time.

Reply:
Just out of interest, I ran a build with and without this imported function. The bundle was 82k smaller without it!

Reply:
![Screenshot 2025-02-05 at 1 44 35 PM](https://github.com/user-attachments/assets/648f2bc5-760a-4916-a22b-e8bf4e50db57)

Reply:
Oh fair point! Will update before I merge